static/js/interactive-map/NewMapConfig.js

```diff
@@ -0,0 +1,215 @@
+import { Coordinate } from './Geo/Coordinate.js';
+import { PinImages } from './PinImages.js';
+import { ClusterPinImages } from './ClusterPinImages.js';
+import { getLanguageForProvider } from './Util/helpers.js';
+import { defaultCenterCoordinate } from './constants.js';
+
+/**
+ * The configuration for the NewMap component.
+ */
+export default class NewMapConfig {
+  /**
+   * @param {Object} rawConfig Configuration to parse
+   */
+  constructor (rawConfig) {
```

Comment:
Could we call this `jsonConfig`? I think that would help someone who shadows this file better understand what's going on.

Reply:
Good point, changed